src/components/base/Select.js

```diff
@@ -34,6 +34,7 @@ export default function Select(props) {
       {props.getLabel && <Label htmlFor={props.name}>{props.getLabel()}</Label>}
       <Input
         id={props.name}
+        data-testid={props.name}
```

Comment:
pareil que pour les boutons et les modals =>
```suggestion
        data-testid={props['data-testid']}
```

Reply:
Idem